modules/images-configuration-registry-mirror.adoc

```diff
@@ -29,276 +29,30 @@ By pulling container images needed by {product-title} and then bringing those im
-  location = "registry.example.com"
-
-  [[registry.mirror]]
-    location = "mirror.example.net/registry-example-com"
-    pull-from-mirror = "digest-only"
-
-[[registry]]
-  prefix = ""
-  location = "registry.example.com/example"
-
-  [[registry.mirror]]
-    location = "mirror.example.net"
-    pull-from-mirror = "digest-only"
-
-[[registry]]
-  prefix = ""
-  location = "registry.example.com/example/myimage"
-
-  [[registry.mirror]]
-    location = "mirror.example.net/image"
-    pull-from-mirror = "digest-only"
-
-[[registry]]
-  prefix = ""
-  location = "registry.redhat.io"
-
-  [[registry.mirror]]
-    location = "mirror.example.com"
-    pull-from-mirror = "digest-only"
-
-[[registry]]
-  prefix = ""
-  location = "registry.redhat.io/openshift4"
-
-  [[registry.mirror]]
-    location = "mirror.example.com/redhat"
-    pull-from-mirror = "digest-only"
-[[registry]]
-  prefix = ""
-  location = "registry.access.redhat.com/ubi9/ubi-minimal"
-  blocked = true <4>
-
-  [[registry.mirror]]
-    location = "example.io/example/ubi-minimal-tag"
-    pull-from-mirror = "tag-only" <5>
-----
-<1> Indicates the repository that is referred to in a pull spec.
-<2> Indicates the mirror for that repository.
-<3> Indicates that the image pull from the mirror is a digest reference image.
-<4> Indicates that the `NeverContactSource` parameter is set for this repository.
-<5> Indicates that the image pull from the mirror is a tag reference image.
-
-.. Pull an image to the node from the source and check if it is resolved by the mirror.
-+
-[source,terminal]
-----
-sh-4.2# podman pull --log-level=debug registry.access.redhat.com/ubi9/ubi-minimal@sha256:5cf...
-----
-
-.Troubleshooting repository mirroring
-
-If the repository mirroring procedure does not work as described, use the following information about how repository mirroring works to help troubleshoot the problem.
+For new clusters, you can use IDMS, ITMS, and ICSP CRs objects as desired. However, using IDMS and ITMS is recommended.
 
-* The first working mirror is used to supply the pulled image.
-* The main registry is only used if no other mirror works.
-* From the system context, the `Insecure` flags are used as fallback.
-* The format of the `/etc/containers/registries.conf` file has changed recently. It is now version 2 and in TOML format.
-* You cannot add the same repository to both an `ImageDigestMirrorSet` and an `ImageTagMirrorSet` object.
+If you upgraded a cluster, any existing ICSP objects remain stable, and both IDMS and ICSP objects are supported. Workloads using ICSP objects continue to function as expected. However, if you want to take advantage of the fallback policies introduced in the IDMS CRs, you can migrate current workloads to IDMS objects by using the `oc adm migrate icsp` command as shown in the *Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring* section that follows. Migrating to IDMS objects does not require a cluster reboot. 
```

Comment:
Under the [Converting ICSP](https://docs.openshift.com/container-platform/4.14/openshift_images/image-configuration.html#images-configuration-registry-mirror-convert_image-configuration), we can add the step 3 mentioning that the customer can remove existing ICSP objects after IDMS objects get rolled out. And This will result in the same registries.conf and hence not result in a reboot.